### PR TITLE
All available API versions made accessible in version column on api d…

### PR DESCRIPTION
**Pull request by radekkaluzik**
…ocs page

https://issues.redhat.com/browse/RHCLOUD-16640

### CODE REVIEW

File: src/Utilities/overviewRows.js [hunk 1]
```diff
@@ -63,14 +65,26 @@ export const rowMapper = (
...
-    version ? `/api/${apiName}` : '',
+    versions ? `/api/${apiName}` : '',
     {
-      title: <Badge>{version}</Badge>,
-      value: version,
+      title: (
+        <Fragment>
+          {versions &&
+            versions.map((version) => (
+              <Link key={version} to={version}>
```
Comment (Hyperkid123) [inline src/Utilities/overviewRows.js:74]: I think you might be missing the `apiName` and the leading `/`. this should most likely look like
```jsx
<Link key={version} to={`/${apiName}/${version}`}>...
```

The same goes for the link above

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

File: src/Utilities/overviewRows.js (hunk 1 above)
Comment (radekkaluzik) [reply src/Utilities/overviewRows.js]: Fixed

**Comment by Hyperkid123** [comment]
@radekkaluzik I've just tried the UI locally. The links are working. Thing is, the v2 open API file does not exist.

**Review by Hyperkid123 — APPROVED** [review]
I am approving as the changes do not introduce any regressions. We should contact the team that requested the feature about the missing open API spec for v2.

**Pull request merged**